frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html

```diff
@@ -1,11 +1,83 @@
-        {{message}}
+    <ng-container *ngIf="titleImagePath; else windowTitleWithoutImage">
+        <div class="title-container with-image-container">
+            <div class="mat-title-img">
+                <img src="{{ titleImagePath }}" />
+            </div>
+            <div class="mat-title-text">
+                <h1 mat-dialog-title>{{ title }}</h1>
+            </div>
+        </div>
+    </ng-container>
+    <ng-template #windowTitleWithoutImage>
+        <div class="title-container">
+            <div class="mat-title-text">
+                <h1 mat-dialog-title>{{ title }}</h1>
+            </div>
+        </div>
+    </ng-template>
+    <div *ngIf="message" mat-dialog-content [align]="'center'" class="message">
+        {{ message }}
     </div>
-    <div mat-dialog-actions [align]="'center'">
-        <button *ngFor="let buttonOptions of buttonsOptions" [ngClass]="buttonOptions.class" mat-flat-button
-                (click)="onClick(buttonOptions.onClickEvent)">{{buttonOptions.label}}</button>
+    <div *ngIf="date || time || duration || meetingName" class="created-meeting-options">
+        <div class="option">
+            <div class="option-name">Date</div>
+            <div class="option-value">{{ date }}</div>
+        </div>
+        <div class="option">
+            <div class="option-name">Time</div>
+            <div class="option-value">{{ time | date: 'HH:mm' }}</div>
+        </div>
+        <div class="option">
+            <div class="option-name">Duration</div>
+            <div class="option-value">{{ duration }} min</div>
+        </div>
+        <div class="option">
+            <div class="option-name">Meeting Name</div>
+            <div class="option-value">{{ meetingName }}</div>
+        </div>
+    </div>
+    <div *ngIf="participants" class="created-meeting-participants">
+        <div class="participants-label">
+            <div class="option-name">Participants: {{ participants.length }}</div>
+        </div>
+        <div class="participants-avatars">
+            <ng-container *ngFor="let participant of participants">
+                <ng-container *ngIf="participant.image; else noImageParticipant">
+                    <span class="avatar"><img src="{{ participant.image }}" /></span>
+                </ng-container>
+                <ng-template #noImageParticipant>
+                    <span class="avatar"><img src="assets/header/avatar-icon.png" /></span>
+                </ng-template>
+            </ng-container>
+            <span class="avatar" *ngIf="participants.length > 10">
+                <div class="left-participants">
+                    <h4>+{{ participants.length - maxVisibleMembers }}</h4>
+                </div>
+            </span>
+        </div>
+    </div>
+    <div *ngIf="location && link" class="created-meeting-options">
+        <div class="option">
+            <div class="option-name">Location</div>
+            <div class="option-value">{{ location }}</div>
```

Comment:
This approach will cause the location type "Google Meet" to be written together.
![image](https://user-images.githubusercontent.com/58224993/187734218-6ffef484-2c53-42ba-b3fe-ed3ca8221856.png)

Reply:
![image](https://user-images.githubusercontent.com/71783287/187744662-7662a8f9-f0fa-4745-92ed-2cd7e0eb4185.png)